src/main/kotlin/net/atos/zac/documentcreation/DocumentCreationService.kt

```diff
@@ -50,11 +55,16 @@ class DocumentCreationService @Inject constructor(
             zaak = documentCreationDataAttended.zaak,
             taskId = documentCreationDataAttended.taskId
         ).let { data ->
-            createSmartDocumentForAttendedFlow(documentCreationDataAttended).let { smartDocument ->
-                smartDocumentsService.createDocumentAttended(
-                    data = data,
-                    smartDocument = smartDocument
-                )
+            try {
+                createSmartDocumentForAttendedFlow(documentCreationDataAttended).let { smartDocument ->
+                    smartDocumentsService.createDocumentAttended(
```

Comment:
similarly here, instead of `let` use `run` and 'cause the `smartDocument` named parameter is used only once when passing it to another parameter named the same, I'd not name this at all, and just use `it`.

Reply:
This means a nested run (two `this`), so we'll have to add labels. Would keep this as unnamed let